iaso/api/logs.py

```diff
@@ -18,10 +18,10 @@
 def has_access_to(user: User, obj: Union[OrgUnit, Instance, models.Model]):
```

Comment:
Changes look good at first look, but I think such an important would deserve its own extensive testing (instead of being quickly tested through the API), this issue would have been a good opportunity to add those. Since it's release day, may I suggest that we release like it is, but create another ticket for the test?